CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 1.1.0
+
+**BUG FIX**
+
+* Fix an issue that was causing terraform-apply failure when ParallelCluster API 3.11.x is used to deploy clusters with login nodes.
```

Comment:
This is a provider's change, not module's.
This changelog should only contain changes to the module.
the change here is to use the provider 1.1.0 in modules examples